homework/style.css

```diff
@@ -1,3 +1,141 @@
+*,
```

Comment:
I may have missed this on my previous reviews but on a mobile device your UI does not look responsive (enough). On an iPhone 6 it would like this, which is already cramped:

![image](https://user-images.githubusercontent.com/2788771/65970202-5c6daf00-e466-11e9-9de7-b1e4667c245c.png)


On an iPhone 5 with its smaller screen it becomes unusable:

![image](https://user-images.githubusercontent.com/2788771/65970299-8aeb8a00-e466-11e9-8052-fd4c9cacc0cf.png)

Reply:
Thank you Jim. It was correct in the previous PR. 
I missed to update the container ID in my css.

I will send the updated version along with the rest of the homework as you suggested.